frontend/src/LineGraph.res

```diff
@@ -211,7 +236,7 @@ let defaultOptions = (
       },
     },
     "series": {
-      "std-dev": {
+      "avg": {
```

Comment:
Why do call this `avg`? IIRC there's a standard deviation function we use to calculate the bands. :)

Reply:
Yes, we do have a standard deviation function that we use to calculate the bands. We display the mean, and the band is one stdDev above and below the mean.  The value shown in the legend, though, is the mean and not the stdDev. may be calling the series "mean" would be better.

Reply:
ah okay, yes that makes sense!